.github/workflows/main.yml

```diff
@@ -155,6 +155,9 @@ jobs:
           path: ~/.cache/coursier
           key: build-${{ matrix.spark_version }}-scala${{ matrix.scala_version }}-java${{ matrix.java_version }}-coursier
 
+      - name : Test - Run spark application for standalone cluster on docker
```

Comment:
Move this test before `Test - Checkout Spark repository`.

It seems we don't need spark code, becasue it's completely docker based